doc/versionHistory.rst

```diff
@@ -2,6 +2,19 @@
 Version History
 ===============
 
+v0.9.0
+------
+
+* Add unit tests for `interruptWEP` command.
+* Add xml 10/11 backward compatible command `interruptWEP`.
+  The command won't be available for xml 10, but CSC will continue to work and automatically support when it is released.
```

Comment:
Please consider to add a ticket to remove the support of xml 10 (to simplify the code) once we are in the xml 11. Thanks!

I just saw this (DM-33401) in `utility.py`. Please help to add this ticket number here as well.